Orbitersdk/samples/ProjectApollo/src_lm/LEM.h

```diff
@@ -1755,6 +1766,16 @@ class LEM : public Payload, public PanelSwitchListener {
 	double vcFreeCamSpeed;
 	double vcFreeCamMaxOffset;
 
+	//
+	// EVA Antenna Handle
+	//
+
+	int EVAAntHandleStatus = false;
```

Comment:
-EVAAntHandleStatus should be initialized in LEM::Init()
-EVAAntHandleProc and meshidxEVAAntHandle seem to be unused variables